tests/unit/-private/column-tree-test.js

```diff
@@ -50,7 +33,6 @@ module('Unit | Private | ColumnTree', function(hooks) {
     tree = ColumnTree.create({
       columns,
       columnMetaCache,
-      rows: generateTree([0, [1, [2, 3], 4, [5, 6]]]),
```

Comment:
This was causing some sort of error, but seems irrelevant to the test, right? A bit of explanation in a comment would be helpful for review.